misc/N and its Double.c

```diff
@@ -0,0 +1,46 @@
+int main()
+{
+    int N,a,i,j,count=0;
+    printf("Enter N=");
```

Comment:
In questions like these we do not give input prompt asking for input, so u can remove all the input prompts everywhere and assume that the user knows what has to be given